railties/lib/rails/generators/rails/authentication/authentication_generator.rb

```diff
@@ -52,6 +52,20 @@ def add_migrations
         generate "migration", "CreateSessions", "user:references ip_address:string user_agent:string", "--force"
       end
 
+      def configure_test_helper
+        inject_into_file "test/test_helper.rb", after: "# Add more helper methods to be used by all tests here...\n" do
+          <<-RUBY
+    def login_as(user)
+      session = user.sessions.create!
+      Current.session = session
+      request = ActionDispatch::Request.new(Rails.application.env_config)
+      cookies = request.cookie_jar
+      cookies.signed[:session_id] = { value: session.id, httponly: true, same_site: :lax }
+    end
```

Comment:
I had success with `logout` locally.

```suggestion
    def login_as(user)
      session = user.sessions.create!
      Current.session = session
      request = ActionDispatch::Request.new(Rails.application.env_config)
      cookies = request.cookie_jar
      cookies.signed[:session_id] = { value: session.id, httponly: true, same_site: :lax }
    end
    
    def logout
      Current.session.destroy
      request = ActionDispatch::Request.new(Rails.application.env_config)
      cookies = request.cookie_jar
      cookies.delete(:session_id)
    end
```